### DISCUSSION TITLE
Derived cryptographic types are obsolete. Use the Create method on the base type instead

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/runtime#63449] [Q&A]
Hello everyone. Today I cloned "runtime" repository and tried to build. I followed build instructions. 
My goal is to reading source code of libraries and tracking the source code on visual studio.
I want to open System.Linq.sln clearly.

First I build CoreCLR with this command "build.cmd -subset clr" and build succeeded. 
Then I wanted to build libraries so I use this command "build.cmd clr+libs -rc Release" but I got some errors which are writes below. 

Could you please help me to clean build. Thanks. 

C:\Users\mrkiyak\source\repos\runtime\src\libraries\System.Security.Cryptography.Algorithms\ref\System.Security.Cryptography.Algorithms.Forwards.cs(10,67): error SYSLIB0021: 'AesManaged' is obsolete: 'Derived cryptographic types are obsolete. Use the Create method on the base type instead.' [C:\Users\mrkiyak\source\repos\runtime\src\libraries\System.Security.Cryptography.Algorithms\ref\System.Security.Cryptography.Algorithms.csproj]
C:\Users\mrkiyak\source\repos\runtime\src\libraries\System.Security.Cryptography.Algorithms\ref\System.Security.Cryptography.Algorithms.Forwards.cs(55,67): error SYSLIB0022: 'Rijndael' is obsolete: 'The Rijndael and RijndaelManaged types are obsolete. Use Aes instead.' [C:\Users\mrkiyak\source\repos\runtime\src\libraries\System.Security.Cryptography.Algorithms\ref\System.Security.Cryptography.Algorithms.csproj]
C:\Users\mrkiyak\source\repos\runtime\src\libraries\System.Security.Cryptography.Algorithms\ref\System.Security.Cryptography.Algorithms.Forwards.cs(56,67): error SYSLIB0022: 'RijndaelManaged' is obsolete: 'The Rijndael and RijndaelManaged types are obsolete. Use Aes instead.' [C:\Users\mrkiyak\source\repos\runtime\src\libraries\System.Security.Cryptography.Algorithms\ref\System.Security.Cryptography.Algorithms.csproj]
C:\Users\mrkiyak\source\repos\runtime\src\libraries\System.Security.Cryptography.Algorithms\ref\System.Security.Cryptography.Algorithms.Forwards.cs(58,67): error SYSLIB0021: 'SHA1Managed' is obsolete: 'Derived cryptographic types are obsolete. Use the Create method on the base type instead.' [C:\Users\mrkiyak\source\repos\runtime\src\libraries\System.Security.Cryptography.Algorithms\ref\System.Security.Cryptography.Algorithms.csproj]
C:\Users\mrkiyak\source\repos\runtime\src\libraries\System.Security.Cryptography.Algorithms\ref\System.Security.Cryptography.Algorithms.Forwards.cs(60,67): error SYSLIB0021: 'SHA256Managed' is obsolete: 'Derived cryptographic types are obsolete. Use the Create method on the base type instead.' [C:\Users\mrkiyak\source\repos\runtime\src\libraries\System.Security.Cryptography.Algorithms\ref\System.Security.Cryptography.Algorithms.csproj]
C:\Users\mrkiyak\source\repos\runtime\src\libraries\System.Security.Cryptography.Algorithms\ref\System.Security.Cryptography.Algorithms.Forwards.cs(62,67): error SYSLIB0021: 'SHA384Managed' is obsolete: 'Derived cryptographic types are obsolete. Use the Create method on the base type instead.' [C:\Users\mrkiyak\source\repos\runtime\src\libraries\System.Security.Cryptography.Algorithms\ref\System.Security.Cryptography.Algorithms.csproj]
C:\Users\mrkiyak\source\repos\runtime\src\libraries\System.Security.Cryptography.Algorithms\ref\System.Security.Cryptography.Algorithms.Forwards.cs(64,67): error SYSLIB0021: 'SHA512Managed' is obsolete: 'Derived cryptographic types are obsolete. Use the Create method on the base type instead.' [C:\Users\mrkiyak\source\repos\runtime\src\libraries\System.Security.Cryptography.Algorithms\ref\System.Security.Cryptography.Algorithms.csproj]

**Comment by kuiperzone** [comment]
var aes = Aes.Create(); // ???

**Comment by huoyaoyuan** [comment]
Looks similar to #63461
